.bazelrc

```diff
@@ -0,0 +1,23 @@
+# 
+#  Copyright 2020 Google LLC
```

Comment:
Super nit: The "Copyright" line seems to have an extra space indentation, making it look like a title. Could you remove the extra space to align all text to the same indentation? (Here and everywhere else.)